README.md

```diff
@@ -94,8 +133,8 @@ python -m pip install --upgrade build
 python -m pip install --upgrade twine
 rm -rf dist
 python -m build
-python -m twine upload dist/*
-```
+python -m twine upload dist/*```
+
```

Comment:
Not sure you meant to make this little change?

Reply:
thanks, that was a merge error.  fixed